swagger.yaml

```diff
@@ -0,0 +1,244 @@
+swagger: "2.0"
+info:
+  description: "discretionary-business-grants API Schema"
+  version: "0.0.1"
+  title: "discretionary-business-grants API"
+host: "discretionarybusinessgrants.hackney.gov.uk"
+basePath: "/api"
+tags:
+  - name: "urls"
+    description: "Secure URL for uploading files"
+  - name: "applications"
+    description: ""
+schemes:
+  - "https"
+paths:
+  /urls:
+    post:
+      tags:
+        - "urls"
+      summary: "Retrieve a new secure URL for file upload"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: "body"
+          in: "body"
+          required: true
+          schema:
+            $ref: "#/definitions/Url"
+      responses:
+        "500":
+          description: "Invalid input"
```

Comment:
A 500 is an internal server error, we want 400 for invalid input

Reply:
We don't have any serverside validation atmo. 400 would imply we have some validation. 

Thoughts?

Reply:
Discussed offline: We will have validation and a 400 is still possible without it.